platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/JsonModelWriter.kt

```diff
@@ -72,6 +72,10 @@ class JsonModelWriter(val writer: Writer) {
         comma()
         property("cacheAction", cacheAction)
         comma()
+        property("cacheActionDescription") {
+            writeStructuredMessage(cacheActionDescription)
+        }
+        comma()
```

Comment:
This is the main behavior change -- exposing an additional field in the CC report model